andela/suyabay
# #116671299 Fix side navigation bar in admin dashboard

#### What does this PR do?

Fix side navigation bar in admin dashboard
#### Description of Task to be completed?

Display the sidebar on the admin dashboard
#### How should this be manually tested?
1. Log in as an administrator,
2. Navigate to the admin dashboard and see sidebar
#### Any background context you want to provide?

Minification of the stylesheets resulted in some errors in main.css.
Upon fixing them, I upset the balance on the admin dashboard page
#### What are the relevant pivotal tracker stories?
#116671299
#### Screenshots (if appropriate)

<img width="1436" alt="screen shot 2016-03-31 at 20 39 03" src="https://cloud.githubusercontent.com/assets/16223627/14185008/aec43fe8-f780-11e5-866a-1907d48b0c20.png">
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
